### DISCUSSION TITLE
'Turn on settings on this panel' - does not work correctly

### DISCUSSION THREAD

**Discussion by vladdoletskiy** [react-page/react-page#1209] [Q&A] [answered]
Hi, there is a problem in your demo which is also there in my app. Button "Turn on settings on this panel" - When you go to the background change panel - and first you turn it off, then you can’t turn it back on. 
https://react-page.github.io/

**Comment by macrozone** [comment, marked as answer]
hi @vladdoletskiy 

thanks, can reproduce. This problem is related to the recent mui 5 update

**Reply by vladdoletskiy** [reply]
Thank you! I do not use mui 5. I use "@material-ui/core": "^5.0.0-alpha.29",

**Reply by zehawki** [reply]
@macrozone possible to get this fix on v4 as well. I see the same thing happening.
